### ISSUE TITLE
Icon does not have accessible-role

### ISSUE THREAD

**Issue by hkshetry** [SAP/ui5-webcomponents#4499]
### **Bug Description**
Icon does not have accessible-role but it has accessible-name

### **Expected Behavior**
In version of 1.0.1 web compoent, `aria-label `is changed to ` accessible-name ` but still ` role ` remains same in Icon which is causing issue to fail automation tests so Icon should  have` accessible-role ` instead of ` role `

Scenarios:
 Its an icon that functions as link in the page’s context. It opens a new page in a new window and hence cannot be a “button” for screen reader users. WCAG also recommends against swapping roles - https://w3c.github.io/aria-practices/#button

### **Context**
- UI5 Web Components version: 1.0.1
- OS/Platform: NA
- Browser: NA
- Affected component: 1.0.1

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactor
- Bussiness impact: A11y Automation Tests are failing

**Comment by ilhan007** [comment]
Closed with https://github.com/SAP/ui5-webcomponents/pull/4548, will be available with with 1.1.0 by 21 Jan
